src/dapp/DAppCreate.tsx

```diff
@@ -0,0 +1,118 @@
+import React, {useEffect} from 'react';
+import {withRouter} from "react-router";
+import {connect} from "react-redux";
+import {Button, Form} from "react-bootstrap";
+import {CreateDAppOfferModel, initDAppCreateOffer} from "./types/CreateDAppOfferModel";
+import {Alert} from 'react-bootstrap';
+import {createDappOffer} from "./functions/create-dappoffer";
+
+const DAppCreate = (props: any) => {
+    const createModel : CreateDAppOfferModel = initDAppCreateOffer;
+    const CreateDApp = async (event : any) => {
+        event.preventDefault();
+        event.stopPropagation();
+        try{
+            var obj = await createDappOffer(createModel);
+           props.history.push("/dappoffer/"+obj.Id);
+        }
+        catch(er){
+            addError(er);
+        }
+    };
+    const [error, setError] = React.useState(<div/>);
```

Comment:
React.useState<JSX.Element>(<></>);
Deze is net wat netter maar ik ga er niet over zeiken

Reply:
goeie maar als ik het hier wijzig moet ik het overal doen. dit is een goed streven om in de toekomst aan te houden.